packages/frontend/src/admin-pages/components/nonprofit/yourPage/tabs/Media.vue

```diff
@@ -259,6 +259,7 @@ export default {
         vue.clearModals()
       }).catch(function (err) {
         vue.clearModals()
+        console.log('ERR', err)
```

Comment:
do we want this to stay here?

Reply:
Nope.

Reply:
I remove that one, and one earlier in the file.